Winium/Winium.Mobile.Common/ConnectionInformation.cs

```diff
@@ -1,10 +1,10 @@
-﻿namespace Winium.StoreApps.Common
+﻿namespace Winium.Mobile.Common
 {
     public class ConnectionInformation
     {
         #region Public Properties
 
-        public const string FileName = ".Winium.StoreApps.ConnectionInformation.json";
+        public const string FileName = ".Winium.Mobile.ConnectionInformation.json";
```

Comment:
I will take a look at this myself. It would be nice to have a gradual transition if it easy to implement. Else we should include some useful message if not connection file was found, so it would be easy to pinpoint the problem of incompatible driver and inner server by end user, who might have messed with versions of inner and outer parts.